
a = input("Enter the first number: ")
b = input("Enter the second number: ")
try:
    print("The addition is: ", int(a)+int(b))
except Exception as y:
    print(y)
print("This code is running properly.")
